pkg/util/errors.go

```diff
@@ -0,0 +1,6 @@
+package util
+
+import "errors"
+
+var ErrCommandNoArgs = errors.New("this command does not take any arguments")
+var ErrUnsupportedPlatform = errors.New("windows is not a supported platform")
```

Comment:
The error isn't windows specific, so the message shouldn't be either
```suggestion
var ErrUnsupportedPlatform = errors.New("unsupported platform")
```